ably-ios/ARTRealtimeChannel.m

```diff
@@ -415,7 +415,7 @@ - (void)detach:(void (^)(ARTErrorInfo * _Nullable))cb {
         case ARTRealtimeChannelDetaching:
         case ARTRealtimeChannelDetached:
             [self.realtime.logger debug:__FILE__ line:__LINE__ message:@"can't detach when not attahed"];
-            if (cb) cb([ARTErrorInfo createWithCode:90000 message:@"Can't detach when not attahed"]);
+            if (cb) cb(nil);
```

Comment:
Can you fix the typo in the already existing code on the line before, `attached` instead of `attahed`?